reports/Oregon.md

```diff
@@ -3715,43 +3722,45 @@ geolocation:
 
 ### Violent arrest made by federal officers | July 28th
 
-A protester is held face-down on the ground by an officer. The protester is punched and struck by another officer with a baton. The protester is then arrested.
+Near 3rd and Salmon, a protester is held face-down on the ground by an officer. The protester is punched and struck by another officer with a baton. The protester is then arrested.
 
 tags: arrest, baton, protester, punch, strike
 
 id: or-portland-192
 
-geolocation: 
+geolocation: 5.5161982, -122.6766199
 
 **Links**
 
 * https://twitter.com/begoniacloak/status/1287785075459756040
+* [Archive Link](https://gateway.temporal.cloud/ipns/2020pb-archive.temporal.cloud/pb2020/or-portland-192.1.mp4)
 
 
 ### Police shine strobe light at protesters, destroy umbrella | July 28th
 
-Footage shows police directing a bright strobe light at protesters and press attempting to film. One officer grabs a protester's umbrella, breaking it.
+Footage shows police neardirecting a bright strobe light at protesters and press attempting to film near 3rd and Main. One officer grabs a protester's umbrella, breaking it.
 
 tags: abuse-of-power, property-destruction, protester
 
 id: or-portland-195
 
-geolocation: 
+geolocation: 45.5156595, -122.6769109
 
 **Links**
 
 * https://twitter.com/begoniacloak/status/1287795370358013952
```

Comment:
This Twitter link has died as well.

Reply:
That's what the archive link is for. So we have tracing to the original poster but the video is still there.